src/Microsoft.AspNetCore.Blazor.Build/Microsoft.AspNetCore.Blazor.Build.nuspec

```diff
@@ -4,7 +4,10 @@
     <id>Microsoft.AspNetCore.Blazor.Build</id>
     <version>$version$</version>
     <authors>Microsoft</authors>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <description>Build mechanism for Blazor applications.</description>
+    <licenseUrl>https://raw.githubusercontent.com/aspnet/Home/2.0.0/LICENSE.txt</licenseUrl>
+    <projectUrl>https://asp.net/</projectUrl>
```

Comment:
Why not the link to this GitHub repository? I always find that very unhelpful when i click on Project Page on nuget just to land on the generic asp.net Homepage.

Reply:
Sounds like a good suggestion to me, but maybe @rynowak knows of some reason why it has to be just `https://asp.net/`. I'm OK with it either way.

Reply:
😆 - Dan and I had the same discussion before I opened this PR. Since we wanted to get this done in a hurry, I just did what the existing packages do. How about it @Eilon - is this allowed to be the github repo URL?

Reply:
I think you should also add the repositoryUrl - https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#repositoryurl (like [this](https://github.com/cake-build/cake/pull/2240/files))

Reply:
The project URL is correct - it's part of ASP.NET. But indeed the *repo* URL should point to this repo.